l10n_fr_department/model/res_partner.py

```diff
@@ -18,33 +20,35 @@ class ResPartner(models.Model):
     @api.depends("zip", "country_id", "country_id.code")
     # If a department code changes, it will have to be manually recomputed
     def _compute_department(self):
-        rcdo = self.env["res.country.department"]
+        def _get_zipcode(p) -> str:
```

Comment:
```suggestion
        def _get_zipcode(partner) -> str:
```

On this method, I recommend renaming `p` to `partner` and `p_zip` to `partner_zip`.
The abbreviation seems unnecessary and makes it more difficult to read IMO